src/MockQueryable/MockQueryable.NSubstitute/NSubstituteExtensions.cs

```diff
@@ -12,9 +13,26 @@ public static class NSubstituteExtensions
             var enumerable = new TestAsyncEnumerable<TEntity>(data);
             ((IAsyncEnumerable<TEntity>)mock).GetEnumerator().Returns(enumerable.GetEnumerator());
             mock.Provider.Returns(enumerable);
-            mock.Expression.Returns(data.Expression);
-            mock.ElementType.Returns(data.ElementType);
-            mock.GetEnumerator().Returns(data.GetEnumerator());
+            mock.Expression.Returns(data?.Expression);
+            mock.ElementType.Returns(data?.ElementType);
+            mock.GetEnumerator().Returns(data?.GetEnumerator());
+            return mock;
+        }
+
+        public static DbSet<TEntity> BuildMockDbSet<TEntity>(this IQueryable<TEntity> data) where TEntity : class
```

Comment:
Could you please also add simple of usage the method to https://github.com/romantitov/MockQueryable/tree/master/src/MockQueryable/MockQueryable.Sample
And update README.md